docs/pre-release/releaseNotes.yml

```diff
@@ -38,6 +38,52 @@ docDescription: >-
 changelog: https://github.com/telepresenceio/telepresence/blob/$branch$/CHANGELOG.md
 
 items:
+  - version: 2.4.10
+    date: "(TBD)"
+    notes:
+      - type: feature
+        title: Application Protocol Strategy
+        body: >-
+          The strategy used when selecting the application protocol for personal intercepts can now be configured using
+          the <code>intercept.appProtocolStrategy</code> in the <code>config.yml</code> file.
+        docs: reference/config/#intercept
+        image: telepresence-2.4.10-intercept-config.png
+      - type: feature
+        title: Helm value for the Application Protocol Strategy
+        body: >-
+          The strategy when selecting the application protocol for personal intercepts in agents injected by the
+          mutating webhook can now be configured using the <code>agentInjector.appProtocolStrategy</code> in the Helm chart.
+        docs: install/helm
+      - type: feature
+        title: New <code>--http-plaintext</code> option
```

Comment:
This doesn't seem to render properly in the `title` element, the `<code>` tag is written in the docs website